src/hotspot/os/posix/os_posix.cpp

```diff
@@ -601,6 +601,14 @@ void os::Posix::print_active_locale(outputStream* st) {
   }
 }
 
+void os::print_jni_name_prefix_on(outputStream* st, int args_size) {
+  // no prefix required, not even "_"
```

Comment:
You could drop the "not even '_'" as there is no context to explain why we might have considered there is a need for it.

Reply:
Hi David, thanks for the review !  I adjusted the comment.
I guess I can push this as trivial with 1 review ?